### DISCUSSION TITLE
104.1 : Open existing file - directory not found ?

### DISCUSSION THREAD

**Discussion by Simon-72** [ClosedXML/ClosedXML#2421] [Q&A]
just updated from 102.3 to 104.1 and not changing anything now when opening an existing .xlsx file it is giving a directory not found error, "XLWorkbook workbook = new XLWorkbook(ReportTempFileName);" when the directory does exist, anybody else had this occur ?

**Comment by nh-tu-z** [comment]
@Simon-72 is this issue solved yet?
